spec/features/borrower_authorization_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.feature "only authorized borrower can create or edit loan requests" do
+  let!(:borrower) { User.create(name: "Richard",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.